### PR TITLE
[tomyumswap] Strategy tomyumswap added

### PR DESCRIPTION
**Pull request by tomyumswap**
Fixes # .

Changes proposed in this pull request:
- New strategy with the name tomyumswap has been added
- Based on the token TOMYUM
- Minimum voting power needed: 0.1
- Voting power and subgraph endpoints have been changed

All test cases are passing

### CODE REVIEW

File: src/strategies/tomyumswap/index.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  if (!results) {
+    return;
+  }
+
+  try {
+    const poolAddresses = results.smartChefs.map((pool) => pool.id);
+    const promises = addresses.map((address) => {
+      return fetchVotingPower(address, blockTag, poolAddresses);
+    }) as ReturnType<typeof fetchVotingPower>[];
+    const votingPowerResults = await Promise.all(promises);
```
Comment (ChaituVR) [inline src/strategies/tomyumswap/index.ts:85]: Hi @tomyumswap sending one request per address won't scale for a large number of votes, please find an alternate way to get all data for all addresses in a single request
https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by tomyumswap** [comment]
Hi Chaitanya

The same code is working for Pancake (strategy name: cake) and AKS
(strategy name: aks) strategy.

**Comment by ChaituVR** [comment]
Probably `cake` strategy needs to be changed too, on aks strategy, it is not doing for all addresses, 🤔 if we don't change, it won't def. scale for larger votings

**Comment by tomyumswap** [comment]
Hi Chaitanya

Pls check the latest [commit](https://github.com/snapshot-labs/snapshot-strategies/pull/176/commits/4c3d9147f03b91d385980409af05480a27f1599b). I have changed the API to include an array of addresses for fetching the voting power. This should fix the scalability issue of the addresses.

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**